go/ql/lib/change-notes/2025-07-15-path-injection-sanitizers.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Remove model `CreateTemp` function, from the `os` package, as a path-injection sink due to proper sanitization by Go. Add check for `os.PathSeparator` in sanitizers for path-injection query.
```

Comment:
Note that it's only the second argument of `CreateTemp` that's no longer a sink.